source/reference/aggregation-variables.txt

```diff
@@ -107,6 +107,18 @@ MongoDB offers the following system variables:
 
      - One of the allowed results of a :pipeline:`$redact` expression.
 
+   * - .. variable:: SEARCH_META 
+
+     - A variable that stores the metadata results of an Atlas Search 
+       query. In a :pipeline:`$project` stage, a field set to the 
```

Comment:
Similar to my comment on this PR: https://github.com/10gen/cloud-docs/pull/2424#pullrequestreview-764761761

I don't know that we want to tie `$$SEARCH_META` directly to project.  It is similar to `$$NOW` and can be accessed a number of ways.